{
  "candidate": "CVE-2021-27923",
  "description": "Pillow before 8.1.1 allows attackers to cause a denial of service (memory consumption) because the reported size of a contained image is not properly checked for an ICO container, and thus an attempted memory allocation can be very large.",
  "notes": {
    "mdeslaur": "while this is mentioned in the 8.1.1 release notes, it doesn't seem to be mentioned in the CHANGES file, and I can't seem to locate the commits that fix this in 8.1.1 vs 8.1.0 This was actually fixed in 8.1.2."
  },
  "priority": "medium",
  "cvss": {
    "nvd": {
      "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
      "score": 7.5,
      "severity": "HIGH"
    }
  },
  "references": [
    "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-27923",
    "https://pillow.readthedocs.io/en/stable/releasenotes/8.1.1.html",
    "https://ubuntu.com/security/notices/USN-4763-1"
  ],
  "assigned_to": "mdeslaur",
  "discovered_by": "Jiayi Lin, Luke Shaffer, Xinran Xie, Akshay Ajayan",
  "public_date": "2021-03-03T09:15:00Z",
  "public_date_at_usn": "2021-03-03T09:15:00Z",
  "packages": [
    {
      "name": "pillow-python2",
      "status": "DNE"
    },
    {
      "name": "python-imaging",
      "status": "DNE"
    },
    {
      "name": "pillow",
      "status": "released",
      "note": "5.1.0-1ubuntu0.5",
      "patches": [
        {
          "source": "upstream",
          "text": "https://github.com/python-pillow/Pillow/commit/480f6819b592d7f07b9a9a52a7656c10bbe07442"
        }
      ]
    }
  ]
}
